File: 0215-kth-largest-element-in-an-array/0215-kth-largest-element-in-an-array.js
var findKthLargest = function(nums, k) {
    const heap = new MaxPriorityQueue();
    for (let i = 0; i < nums.length; i++) {
        heap.enqueue(nums[i]);
    }
    let ret = 0;
    for (let i = 0; i < k; i++) {
        ret = heap.dequeue();
    }
    return ret.element;
};

File: 0215-kth-largest-element-in-an-array/0215-kth-largest-element-in-an-array.py
class Solution:
    def findKthLargest(self, nums: List[int], k: int) -> int:
        
        nums = [-num for num in nums]
        heapify(nums)
        ret = float("inf")
        for _ in range(k):
            ret = -heappop(nums)
        return ret
